Add --format=json to inspect and list commands (#2936)
docker/cli: docs/reference/commandline/node_ls.md

35o6tiywb700jesrt3dmllaza: swarm-worker1 Needs Rotation
```

> To print all the nodes with json output, use `json` directive:

Reviewer: Can you add a blank line between the text and the code block? Also perhaps;

```suggestion
To print all nodes in JSON format, use the `json` directive:

```

Reviewer: Perhaps also for the other docs that were added (JSON uppercase, "_the_ `json` directive"

> ```bash

Reviewer: can you change this to `console`? (I'll be updating other occurrences, so I could do that at the same time if needed; some background why `console` here: https://github.com/docker/docker.github.io/pull/12425)

Reviewer: Also for some of the other docs that were added

Author: Sure 👍

$ docker node ls --format json

> {"Availability":"Active","EngineVersion":"20.10.5","Hostname":"docker-desktop","ID":"k8f4w7qtzpj5sqzclcqafw35g","ManagerStatus":"Leader","Self":true,"Status":"Ready","TLSStatus":"Ready"}

Reviewer: Looks like we're (already) not consistent with pretty/non-pretty printing JSON 🤔
